core/templates/dev/head/css/oppia.css

```diff
@@ -3684,6 +3684,53 @@ md-card.preview-conversation-skin-supplemental-card {
   font-size: 16px;
 }
 
+@media(max-width: 500px) {
+  .oppia-dashboard-container {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .oppia-dashboard-container .dashboard-hide-mobile {
+    display: none !important;
```

Comment:
Is the "important" necessary here. It's better not to use it unless necessary, can be inconvenient or confusing for people who work on this later down the line.